### PR TITLE
[14.0][ADD] account_payment_order_void_line

### PR DESCRIPTION
**Pull request by Murtaza-OSI**

### CODE REVIEW

**Comment by max3903** [comment]
@Murtaza-OSI 

```
<string>:1: (WARNING/2) Title overline too short.

=====================
Account Payment Order Void Lines
=====================
2021-03-07 21:40:42,461 218 WARNING odoodb odoo.addons.base.models.ir_module: docutils' system message present: <system_message level="2" line="1" source="<string>" type="WARNING"><paragraph>Title overline too short.</paragraph><literal_block xml:space="preserve">=====================
Account Payment Order Void Lines
=====================</literal_block></system_message>
```

**Comment by pedrobaeza** [comment]
Please avoid plurals in module name.

File: account_payment_order_void_lines/README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
```
Comment (dreispt) [inline account_payment_order_void_lines/README.rst]: Can we instead use the `readme/` subdirectory with readme fragments?

**Review by dreispt — CHANGES_REQUESTED** [review]
[ ]  Change module name to account_payment_order_void_lines
[ ] Add readme/ subdirectory with readme fragments

File: account_payment_order_void_line/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+This module is used to cancel the payment line.
```
Comment (dreispt) [inline account_payment_order_void_line/readme/USAGE.rst:1]: How?
Can you describe the steps to use or test this?

**Review by dreispt — COMMENTED** [review]
Please remove the OSI icon, OCA modules can't be company branded.

**Review by dreispt — APPROVED** [review]

**Pull request closed without merging**